### PR TITLE
Dom -Manipulation

### PR DESCRIPTION
**Pull request by Yoz21**
In this project, with the help of my coding partner, I tried to put some dom-manipulation on the hamburger part of the mobile view.

### CODE REVIEW

**Review by leolpaz — COMMENTED** [review]
Hi @Yoz21 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses the incorrect type of linter for this project, as per the requirement:
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by lily-coder — COMMENTED** [review]
Hi @Yoz21 and @Kayonga99 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

As the previous reviewer mentioned, You have submitted a project, that has no JavaScript linters. To set up the linters, please follow this [link](https://github.com/microverseinc/linters-config/tree/master/javascript). Add the [.eslintrc.json](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) file and also modify the `.github/workflows/linters.yml` file by adding this section of Js linters under `jobs:`

```
eslint:
    name: ESLint
    runs-on: ubuntu-18.04
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: "12.x"
      - name: Setup ESLint
        run: |
          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.eslintrc.json
      - name: ESLint Report
        run: npx eslint .
```

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -351,5 +354,6 @@ <h3 class="best">
         </ul>
       </div>
     </section>
+    <script src="main.js"></script>
   </body>
 </html>
```
Comment (brytebee) [inline index.html:359]: - [ ] **Optional:** Please add a screenshot of your project. https://github.com/Yoz21/Portfolio/blob/ab8a1a282b62b70bbe824f5bc6fb03299be56b25/README.md

File: index.html [hunk 2]
```diff
@@ -25,13 +25,16 @@
...
+        <img id="btn" class="mob-img" src="icons/menu.png" alt="menu" />
         <div class="desk-head">
           <ul class="desk-li">
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contacts</li>
-            <li>
+            <li class="close">
+              <img id="close" src="icons/Icon.png" alt="" />
+            </li>
```
Comment (brytebee) [inline index.html:33]: - [ ] **Required:** You have done an amazing job. However, I have noticed that the close "X" shows up on the desktop view. please fix this.
- [ ] **Optional:** There are some sections such as the header, projects and contact that have overflowing empty spaces that have not been fixed, please see the attached gif for more.

x overflowing

File: index.html [hunk 3]
```diff
@@ -25,13 +25,16 @@
...
+            <li class="common-li" id="portfolio">Portfolio</li>
+            <li class="common-li" id="about">About</li>
+            <li class="common-li" id="contact">Contacts</li>
```
Comment (brytebee) [inline index.html:36]: - [ ] **Required:** Excellent job implementing this 👏. To ensure it is completely similar  to your chosen template, please add the following:

-   Change the `font-family` from Times new roman to Inter, popkin etc as used in your project.
- Align the text to the left as shown in the Figma Template also attached below.
- Add some padding to the right to push the `hr` away from the screen.

Your Implementation | Figma Template
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi Team👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- use of js best practices✔️
- good implementation of event listeners✔️
- good use of JS to manipulate DOM and call CSS classes✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Yoz21) [reply index.html:36]: Thanks for all the comments and suggestions.

**Review by andersonlebon — APPROVED** [review]
## Hi @Yoz21 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by elyor-doniyorov** [comment]
### Hi @Yoz21, I think you continue with your next project. This project is already approved

**Pull request merged**